addons/udes_stock_package_move/tests/test_change_package_location.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from odoo.exceptions import UserError
+from addons.udes_stock.tests import common
+
+
+class TestChangePackageLocation(common.BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestChangePackageLocation, cls).setUpClass()
+
+        Package = cls.env["stock.quant.package"]
+
+        cls.empty_package = Package.create({})
+        cls.package_1 = cls.create_package(cls.test_stock_location_01)
+        cls.package_2 = cls.create_package(cls.test_stock_location_01)
+        cls.package_3 = cls.create_package(cls.test_stock_location_01)
+        cls.package_4 = cls.create_package(cls.test_stock_location_02)
+        cls.packages = cls.package_1 | cls.package_2 | cls.package_3 | cls.package_4
+
+    @classmethod
+    def create_package(cls, location):
+        """
+        Create a package containing apples and bananas at the given location
+        """
+        Package = cls.env["stock.quant.package"]
+
+        package = Package.create({})
+        cls.create_quant(cls.apple.id, location.id, 10, package_id=package.id)
+        cls.create_quant(cls.banana.id, location.id, 10, package_id=package.id)
+        return package
+
+    def complete_picking(self, picking):
+        """ Marks a picking and all its moves as done """
+        for move_line in picking.move_line_ids:
+            move_line.qty_done = move_line.product_uom_qty
+        picking.action_done()
+
+    def test01_move_empty_package(self):
+        """ Test that moving an empty package fails """
+        ChangePackageLocation = self.env["change_package_location"]
```

Comment:
We can store this in `self.ChangePackageLocation` at setupClass, since it is going to be used by all the tets.

Reply:
In fact, we can move it into a helper function since we are always doing:
```
ChangePackageLocation = self.env["change_package_location"]
wizard = ChangePackageLocation.create({})
wizard = wizard.with_context(active_ids=XXX)
```